Bejebeje.Api/Controllers/ArtistImagesController.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Bejebeje.Api.Controllers
+{
+  using System;
+  using System.Threading.Tasks;
+  using Bejebeje.Common.Exceptions;
+  using Bejebeje.Common.Extensions;
+  using Bejebeje.Services.Services.Interfaces;
+  using Microsoft.AspNetCore.Mvc;
+  using Microsoft.Extensions.Logging;
+
+  public class ArtistImagesController : ControllerBase
+  {
+    private readonly IImagesService imagesService;
+
+    private readonly ILogger logger;
+
+    public ArtistImagesController(
+      IImagesService imagesService,
+      ILogger<ArtistImagesController> logger)
+    {
+      this.imagesService = imagesService;
+      this.logger = logger;
+    }
+
+    [Route("artists/{artistSlug}/image")]
+    public async Task<IActionResult> Get(string artistSlug)
+    {
+      if (artistSlug == null)
+      {
+        throw new ArgumentException(nameof(artistSlug));
```

Comment:
imo this shouldn't be an exception, just a redirect to home with a message or something would be better. cause this will send the user to the exception page. (of course you can configure your Exception Page to redirect to Home with some sort of error, but idk your configuration)